docs/guide/start/policies-cli.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Getting Started - Policies
+title_in_menu: Policies
+layout: website-normal
+menu-parent: index-cli.md
+---
+
+
+
+
+## A Clustered Example
+
+We'll now look at a more complex example that better shows the capabilities of Brooklyn. 
+
+We'll start by deploying an application via YAML blueprint consisting of the following layers.
+
+- A dynamically scalable Web App Cluster
+- A MySQL DB
+
+
+Copy the blueprint below into a text file, "mycluster.yaml", in your workspace, but *before* you create an application 
+with it, again modify the YAML to specify the location where the application will be deployed.  
+You will need at least five machines for this example, one for the DB, and four for the tomcats 
+(but you can reduce this by changing the "maxPoolSize" below.
+
+{% highlight yaml %}
+name: cluster
+
+location:
+  tbd
```

Comment:
Thoughts on changing this to a note reminding the reader to fill in the location details? Or the ec2 or softlayer ABCDE... style?